gwpy/io/datafind.py

```diff
@@ -309,7 +311,6 @@ def on_tape(*files):
 
 def _choose_connection(**datafind_kw):
     if os.getenv('LIGO_DATAFIND_SERVER') or datafind_kw.get('host'):
-        from gwdatafind import connect
         return connect(**datafind_kw)
```

Comment:
@duncanmmacleod, methinks this line will have to change to the following?

```suggestion
         return gwdatafind.connect(**datafind_kw)
```